### PR TITLE
Change mvn install plugin and install utility to use new resolve as se…

### PR DESCRIPTION
**Pull request by TayyabDev**

### CODE REVIEW

**Comment by TayyabDev** [comment]
#build

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ResolveDirector.java [hunk 1]
```diff
@@ -448,15 +448,29 @@ void resolve(String feature, File esaFile, String toExtension) throws InstallExc
         try {
             if (downloadOption == DownloadOption.all || downloadOption == DownloadOption.none) {
                 resolver = new RepositoryResolver(productDefinitions, Collections.<ProvisioningFeatureDefinition> emptySet(), Collections.<IFixInfo> emptySet(), loginInfo);
-                installResources = resolver.resolve(featureNamesProcessed);
+                if (InstallUtils.getIsServerXmlInstall()) {
+                    System.out.println("Using new resolveAsSet API");
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ResolveDirector.java]: Change system out to debug log

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ResolveDirector.java [hunk 2]
```diff
@@ -448,15 +448,29 @@ void resolve(String feature, File esaFile, String toExtension) throws InstallExc
...
+                    installResources = resolver.resolveAsSet(featureNamesProcessed); // use new api
+                } else {
+                    System.out.println("Using old resolve api");
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ResolveDirector.java]: Change system out to debug log

**Comment by alvinso** [comment]
Do we need to add conditions to line 654 in ResolveDirector.java

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
The previous build https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_7Jx70M_7Eemo7shcvMe-cQ
failed due to unrelated fat failures (jsonb fat).
The install tests ran and passed.

**Comment by TayyabDev** [comment]
#build

**Pull request merged**